lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -381,7 +382,9 @@ class DemographicsCollectionModal extends React.Component {
                     </label>
                     <select
                       className="form-control"
+                      autoFocus
                       onChange={this.handleSelectChange}
+                      key="self-education"
```

Comment:
Question -- why did we add a `key` attribute here? We didn't add them to any of the other items.

Reply:
For some reason react was giving this a duplicate key to another child, so manually setting it lets the auto-focus work properly.